src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -56,6 +56,14 @@ const serveFormReact =
         reactFrontendPath,
       },
     })
+    let reactHtml = readFileSync(path.join(reactFrontendPath, 'index.html'), {
+      encoding: 'utf8',
+    })
+    reactHtml = reactHtml
+      // there are multiple __OG_TITLE__ and we are unable to use string.replaceAll so we use regexp
+      .replace(new RegExp('__OG_TITLE__', 'g'), 'FormSG')
+      .replace('__OG_DESCRIPTION__', 'Form Manager for Government')
+
```

Comment:
should we move this block down to after the public form conditional so we do not do double work?

Reply:
yes

Reply:
[9b407b1](https://github.com/opengovsg/FormSG/pull/5272/commits/9b407b152ac4c2f3da77c6257c3cb4f211d0c3a6)